src/modules/DropManager/DropManager.tsx

```diff
@@ -0,0 +1,174 @@
+          <Button size="sm" variant="icon" onClick={onDeleteClick}>
+            <DeleteIcon color="red" />
+          </Button>
+        </Td>
+      </Tr>
+    ));
+
+  const getMobileTableBody = () =>
+    data.map((drop) => (
+      <Tr key={drop.id}>
+        <Td>
+          <VStack align="flex-start" spacing="2">
+            {tableColumns.map((column) => (
+              <Box key={`${drop.id}-${column.title}`}>{column.selector(drop)}</Box>
+            ))}
+          </VStack>
+        </Td>
+        <Td display="flex" justifyContent="right">
+          <Button mr="1" size="sm" variant="icon" onClick={onCopyClick}>
+            <CopyIcon />
+          </Button>
+          <Button size="sm" variant="icon" onClick={onDeleteClick}>
+            <DeleteIcon color="red" />
+          </Button>
+        </Td>
+      </Tr>
+    ));
+
+  return (
+    <Box px="1" py={{ base: '3.25rem', md: '5rem' }}>
+      <Breadcrumbs items={breadcrumbItems} />
+      <Stack direction={{ base: 'column', md: 'row' }}>
+        {/* Left Section */}
+        <Box flexGrow="1">
+          <Stack
+            direction={{ base: 'column', md: 'row' }}
+            pb={{ base: '4', md: '6' }}
+            pt={{ base: '4', md: '10' }}
+            spacing={{ base: '4', md: '3.125rem' }}
+          >
+            {/* Drop name */}
+            <Stack maxW={{ base: 'full', md: '22.5rem' }}>
+              <Text color="gray.800">Drop name</Text>
+              <Heading>Star Invader 3</Heading>
+            </Stack>
+
+            {/* Drops claimed */}
+            <Stack maxW={{ base: 'full', md: '22.5rem' }}>
+              <Text color="gray.800">{claimedHeaderText}</Text>
+              <Heading>{claimedText}</Heading>
+            </Stack>
+          </Stack>
+          <Text>Track link status and export to them to CSV for use in email campaigns here.</Text>
+        </Box>
+
+        {/* Right Section */}
+        <HStack alignItems="end" justify="center" mt="1rem !important">
+          <Button variant="secondary" w={{ base: '100%', sm: 'initial' }}>
+            Cancel all
+          </Button>
+          <Button variant="secondary" w={{ base: '100%', sm: 'initial' }}>
+            Export .CSV
+          </Button>
+        </HStack>
+      </Stack>
+      <Box>
+        {/* Desktop Table - TODO: Refactor Table component to include mobile version */}
+
+        <Show above="sm">
+          <TableContainer>
+            <Table mt="6">
+              {showColumns && (
+                <Thead>
+                  <Tr>
+                    {tableColumns.map((col) => (
+                      <Th key={col.title}>{col.title}</Th>
+                    ))}
+                    {/* Actions header */}
+                    <Th></Th>
+                  </Tr>
+                </Thead>
+              )}
+              <Tbody>{getDesktopTableBody()}</Tbody>
+            </Table>
+          </TableContainer>
+        </Show>
+
+        {/* Mobile table */}
+        <Show below="sm">
```

Comment:
let's standardise the `props`! can we use `<Hide .../>` instead for this

Reply:
Can justify why use `Hide` and `above` for standardisation? For me using `Show below/above` is easy to understand

Reply:
similar to tailwind concept of simplifying breakpoints to always uses (> greater than)
so when it s 'sm', dev will always know that it refers to size 'sm' and above.

Reply:
okay let's go with that